app/res/xml/preferences_developer_pin.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
+    <EditTextPreference
+        android:dialogTitle="Enter Access Pin"
```

Comment:
My guess is that Saket may suggest that this PIN use PW bubbles (since admins enter it in front of users). No strong preference on whether we should oblige on that,